src/extension/background.re

```diff
@@ -72,6 +72,35 @@ Chrome.ContextMenus.create {
   "onclick": fun _ tab => toggleConversion tab##id
 };
 
+let convertExt ext => switch ext {
```

Comment:
This could be shortened to `let convertExt = fun | "ml" => "mli" | ...`

Also, how about the names `mapExtension` or `correspondingExtension` instead?